lib/v2/datasource/remote/api/guardians_repository.dart

```diff
@@ -69,7 +69,7 @@ class GuardiansRepository extends EosRepository with NetworkRepository {
         ..data = {
           'user_account': accountName,
           'guardian_accounts': guardians,
-          'time_delay_sec': const Duration(days: 1).inSeconds,
+          'time_delay_sec': const Duration(minutes: 3).inSeconds,
```

Comment:
THIS MUST BE CHANGED BACK TO 24 hours. 

DO NOT MERGE THIS PR until this is reverted

Reply:
for testing purposes we set it to 3 min